ml-agents/mlagents/trainers/learn.py

```diff
@@ -99,6 +107,8 @@ def run_training(
         lesson,
         run_seed,
         fast_simulation,
+        sampler_manager,
+        resampling_interval,
```

Comment:
is resampling duration the same as resampling interval ? The naming needs to be consistent if this is the case.